mkdocs/docs/HPC/multi_job_submission.md

```diff
@@ -138,6 +138,14 @@ program will now be run for all 100 parameter instances -- 8
 concurrently -- until all computations are done. A computation for such
 a parameter instance is called a work item in Worker parlance.
 
+!!! warning
+    When you want to submit a job on a cluster that is not the default cluster. You need to use:
+    <pre><code>$ <b>module swap env/slurm/<submit cluster\></b>
```

Comment:
Don't use `<...>`, it will confuse people.

Do something like:

```
For example, to submit a worker job to the [donphan debug cluster](...) from the login nodes, use:
...

Reply:
Changed this everywhere to an example (donphan).